Fixed grammatical typo (#2499)
javascript-tutorial/en.javascript.info: 1-js/04-object-basics/06-constructor-new/article.md

Using constructor functions to create objects gives a great deal of flexibility. The constructor function may have parameters that define how to construct the object, and what to put in it.

> Of course, we can add `this` to, not only properties, but methods as well.

Reviewer: but we add the stuff "to this" 
just asking, not native speaker, I dont know where to put "as well"

```suggestion
Of course, we can add not only properties, but methods as well, to `this` .
```
?

> Of course, we can add `this` to, not only properties, but methods as well.

Reviewer: but we add the stuff "to this"
just asking, not native speaker 

```suggestion
Of course, we can add not only properties, but also methods to `this`.
```

Author: Yes the edit you made would work fine as well.

Author: Fixed grammatical typo
